smartsim/entity/application.py

```diff
@@ -247,7 +247,7 @@ def attached_files_table(self) -> str:
         :returns: String version of table
         """
         if not self.files:
```

Comment:
It won't let me comment, but for Application.exe setter:  should we have a change that exe is not “”

Reply:
I can do that

Reply:
Make sure to add this check in the exe property setter as well